### ISSUE TITLE
Make updated draft of Figure 1

### ISSUE THREAD

**Issue by sonyahanson** [choderalab/fluorescence-assay-manuscript#5]
My To Do's here are:
- Move titles to outside rows and columns, like here: http://stackoverflow.com/questions/25812255/row-and-column-headers-in-matplotlibs-subplots
- Add in images of ligands as the row ‘titles’ using this: http://stackoverflow.com/questions/3765056/combine-picture-and-plot-with-python-matplotlib
- Make colorbars as generated by python like here: http://stackoverflow.com/questions/13784201/matplotlib-2-subplots-1-colorbar
- Make it so that we have a choice about which section is being plotted

Adding your To Do's:
- Make figure vector graphic: e.g. PDF, would EPS do?
- Increase text size (moving the titles to rows and columns rather than for individual frame will automatically make this better)

**Comment by jchodera** [comment]
Do we need to remeasure some of [the data](https://github.com/choderalab/fluorescence-assay-manuscript/blob/master/figures/fig1/Figure1.png)? Some of these plots look like there are significant artifacts.

**Comment by sonyahanson** [comment]
The current idea for this first figure is basically this, but with four ligands instead of two:

![spectraassay-v3](https://cloud.githubusercontent.com/assets/6333823/20394886/11a01536-acaf-11e6-8e4d-1c192b2eef2f.png)

I'm curious if we instead want to include more of the spectra, like this?

![fret_effect_proteinfl](https://cloud.githubusercontent.com/assets/6333823/20394847/ee04895e-acae-11e6-9d3f-91397e141888.png)

**Comment by sonyahanson** [comment]
Started making the figure with the inset of the zoom in on the 480 nm fluorescence increase, and  I'm not that happy with it, so started looking into plotting on a log scale:

![logplot_spectra](https://cloud.githubusercontent.com/assets/6333823/20682584/92826828-b576-11e6-97c2-7d7d45b4731d.png)

**Comment by sonyahanson** [comment]
This gives us a figure that would look like this:

![spectra-v5](https://cloud.githubusercontent.com/assets/6333823/20687081/1638f2b0-b589-11e6-82a2-363bfb754120.png)

**Comment by jchodera** [comment]
Are 340 nm and 480 nm curves swapped in that inset? It looks like the Trp fluorescence is going *up* with increasing [L], which seems wrong.

**Comment by jchodera** [comment]
I just noticed that the ligand-only fluorescence for Gefinitib:Src is very different from gefitinib:{Abl,p38}.

**Comment by sonyahanson** [comment]
Yeah, good point on the Gefitinib:Src results, will need to cross check with other experiments, maybe use another dataset for this figure.

Fixed the swapping of the 340/480 figures, and added more recent result for funsies.

![spectra-v6](https://cloud.githubusercontent.com/assets/6333823/20904261/c2931a58-bb0c-11e6-831f-0068762da590.png)
